jetty-ee9/jetty-ee9-nested/src/main/java/org/eclipse/jetty/ee9/nested/ContextHandler.java

```diff
@@ -2861,6 +2865,44 @@ public APIContext getAPIContext()
             {
                 return _apiContext;
             }
+
+            @Override
+            public Object getAttribute(String name)
```

Comment:
I think we also need to update the attribute name set method for these ones.

Reply:
I have now done this for both EE9 and EE10 code.